android/src/main/java/com/reactnativekeyboardcontroller/views/overlay/JSPointerDispatcherCompat.kt

```diff
@@ -0,0 +1,36 @@
+package com.reactnativekeyboardcontroller.views.overlay
+
+import android.view.MotionEvent
+import android.view.ViewGroup
+import com.facebook.react.uimanager.JSPointerDispatcher
+import com.facebook.react.uimanager.events.EventDispatcher
+
+/**
+ * Compat layer for `JSPointerDispatcher` interface for RN < 0.72
+ */
+class JSPointerDispatcherCompat(
+  private val viewGroup: ViewGroup,
+) : JSPointerDispatcher(viewGroup) {
+  fun handleMotionEventCompat(
+    event: MotionEvent?,
+    eventDispatcher: EventDispatcher?,
+    isCapture: Boolean,
+  ) {
+    val method =
+      try {
+        JSPointerDispatcher::class.java.getMethod(
```

Comment:
I would add a comment which RN check we perform here (so it's always clear it's RN > 0.72 code) as first line in `try` block